_deployment/tests/alexa.sh

```diff
@@ -19,12 +20,13 @@ check_rank () {
     cmd="ruby alexa.rb ${domain}"
     $cmd
 
-    # Get exit code from the ruby script
-    status=$?
+    if [ $? -ne 0 ]; then
+      res=1
+    fi
```

Comment:
Shouldn't this work? (And removing `$cmd` on L21)
```suggestion
   $cmd || { res=res+1 }
```

Reply:
Adding a number to a variable would be something like "res=$(($res+1))". That would also mean that the return code is the number of errors here, which is rather unusual for shell scripts. Maybe the worst result should be returned instead.

Reply:
Updated

Reply:
And the `cmd` variable is not needed either...

Reply:
Are you sure? $domain might be the latest run command otherwise 🤔

Reply:
How would `$domain` be executed? It's the argument for `alexa.rb`.